powerline/renderers/tmux.py

```diff
@@ -63,7 +63,7 @@ def get_segment_info(self, segment_info, mode):
 		if segment_info:
 			r.update(segment_info)
 		if 'pane_id' in r:
-			varname = 'TMUX_PWD_' + r['pane_id'].lstrip('%')
+			varname = 'TMUX_PWD_' + r['pane_id'].lstrip('%').lstrip(' ')
```

Comment:
In the bug this is meant to work around, the spaces come first, *before* the percent sign. This code strips spaces second. Best would probably be to remove them both simultaneously: `lstrip('% ')`.